Lib9c/Model/Item/Equipment.cs

```diff
@@ -6,15 +6,26 @@
 using Nekoyume.Model.Stat;
 using Nekoyume.Model.State;
 using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
 
 namespace Nekoyume.Model.Item
 {
     [Serializable]
     public class Equipment : ItemUsable, IEquippableItem
     {
         // FIXME: Whether the equipment is equipped or not has no asset value and must be removed from the state.
-        public bool equipped = false;
+        public bool equipped;
         public int level;
+        public int optionCountFromCombination;
+        public readonly int RequiredCharacterLevel;
+
+        /// <summary>
+        /// Do not use in Lib9c. This property only for UI.
+        /// </summary>
+        public int OptionCountFromCombinationForUI => optionCountFromCombination > 0
```

Comment:
If this property used only in unity, why not implemented in unity?

Reply:
Oh, that's make sense. I'll implement this in other place as extension.